content/docs/iac/concepts/resources/providers.md

```diff
@@ -50,7 +50,42 @@ For example, to generate a local SDK for the [`hashicorp/random` provider](https
 $ pulumi package add terraform-provider hashicorp/random
 ```
 
+{{% notes type="tip" %}}
 In order to make sure Pulumi users are aware of the Any Terraform Provider's capabilities, Pulumi has included select, popular providers that can be consumed in Pulumi via the Any Terraform Provider in the Pulumi Registry, such as [The Honeycomb provider](/registry/packages/honeycombio/).
+{{% /notes %}}
+
+The generated SDK will include a `.gitignore` so it can be safely committed to version control without including all of the SDK's dependencies. The SDK contains instructions to download the provider binary to a shared (and cached) location on your local system outside of the working directory - the binary is not committed to version control.
```

Comment:
"instructions" feels like something the user has to do. Maybe "metadata" or rephrase entirely?

Reply:
```suggestion
The generated SDK will include a `.gitignore` so it can be safely committed to version control without including all of the SDK's dependencies. The SDK installation process also downloads the provider binary to a shared location on your local system outside of the working directory. The binary is cached, so it will not need to be downloaded more than once, and is not committed to version control.
```